js/sales.js

```diff
@@ -11,26 +13,30 @@ function Store(location, maxCust, minCust, avgSalesPerCust){
   this.maxCust = maxCust;
   this.minCust = minCust;
   this.avgSalesPerCust = avgSalesPerCust;
+  stores.push(this);
 }
 
 // switch statement to map location property to proper readable version to display on table
 Store.prototype.mapLocation = function() {
```

Comment:
This method is completely unnecessary. Just pass in the actual name of the store when you construct it.
That is, replace
```javascript
new Store('downtown', 65, 23, 6.3);
```
with
```javascript
new Store('1st and Pike', 65, 23, 6.3);
```